RLTSquare/CustomEmailSender/view/frontend/email/test_email_template.html

```diff
@@ -0,0 +1,5 @@
+<!--@subject {{trans "Test Email Template"}} @-->
+{{template config_path="design/email/header_template"}}
+<H2>Welcome to Our Store</H2>
+
```

Comment:
Properly format/indent your code - there is no space between header_template and welcome message, while there is single space between welcome and footer_template. Not a technical problem, but consistency / formatting one.